### DISCUSSION TITLE
Installing sitepackage on Typo3 composer installation

### DISCUSSION THREAD

**Discussion by mirmnk** [benjaminkott/bootstrap_package#1175] [Q&A]
Hello,

spent several hours trying to add generated sitepackage as local extension to typo3 composer installation.

1.  created folder "extensions" inside installation root folder.
2. Added repositories [ { "type": "path", "url": "extensions/*" } ] to composer.json located in installation root folder 
3. Created dir extensions/company-sitepackage
4. extracted sitepackage .zip there
5. composer require company/my-sample-sitepackage (from root folder)

Getting:

 Could not find a version of package company/sitepackage matching your minimum-stability (stable). Require it with
   an explicit version constraint allowing its desired stability.

If I run:
composer require company/my-sample-sitepackage@dev

Getting:

 The "https://packagist.org/providers/companyt/sitepackage@dev.json" file could not be downloaded (HTTP/1.1 404 Not
   Found)

I am stuck, any help will be appreciated!

**Comment by LRWphil** [comment]
maybe `:` missing?
` composer require company/my-sample-sitepackage:@dev`


maybe local missing?
```
"repositories": {
        "local": {
            "type": "path",
            "url": "./extensions/*"
        }
    }
```
